file.go

```diff
@@ -1,37 +1,127 @@
 package jcsv
 
-import "os"
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+)
 
 type file struct {
-	// TODO: define variables if needed
+	jsonData []uint8
```

Comment:
What is the reason for using `[]uint8`?

Reply:
The smallest accessible memory unit in GO is byte/uint8, and also data returned after parsing json is also of type byte.